src/chrome/content/rules/FAZ.xml

```diff
@@ -1,6 +1,6 @@
 <!--	See FAZ-mismatches.xml also
 -->
-<ruleset name="FAZ (broken)" >
+<ruleset name="FAZ (partial)" >
   <target host="faz.de"/>
   <target host="*.faz.de"/>
   <target host="faz-institute.de"/>
```

Comment:
I just fixed the typos in the file, so the regex match again and create "valid" urls.
I should go over the whole file instead i guess